app/assets/javascripts/mountain_view.js.erb

```diff
@@ -1,5 +1,7 @@
-<% depend_on Rails.root.join('app', 'components').to_s %>
-<% Dir.glob(Rails.root.join('app', 'components', '*')) do |component_dir|
-  component = File.basename component_dir
-  require_asset "#{component}/#{component}"
-end %>
+<% if Dir.exists?(Rails.root.join("app", "components") ) %>
+  <% depend_on Rails.root.join('app', 'components').to_s %>
```

Comment:
I think this line should be above the `if`, so we mark this file depends on that directory, and when the first component is added, it automatically pick up the changes. Same thing with the `css.erb`.

I'm not 100% sure about this though. So it may be better to test it—starting with an empty app, and then adding the first component.